src/get-dependencies-from-document.ts

```diff
@@ -40,14 +53,37 @@ function collectScriptsAndStyles(tree: DocumentDescriptor): DocumentDeps {
   };
 }
 
+/**
+ * Returns a collection of all local dependencies from a DocumentDescriptor
+ * object, ignoring any remote dependencies. Because HTML imports can have
+ * script and style dependencies of their own, this will recursively call
+ * itself down the import tree to collect all dependencies.
+ */
 export function getDependenciesFromDocument(descriptor: DocumentDescriptor, dir: string): DocumentDeps {
   let allHtmlDeps: string[] = [];
   let allScriptDeps = new Set<string>();
   let allStyleDeps = new Set<string>();
-
   let deps: DocumentDeps = collectScriptsAndStyles(descriptor);
-  deps.scripts.forEach((s) => allScriptDeps.add(posixPath.join(dir, s)));
-  deps.styles.forEach((s) => allStyleDeps.add(posixPath.join(dir, s)));
+
+  // Collect all script dependencies
+  deps.scripts.forEach((scriptDep) => {
+    if (isDependencyRemote(scriptDep)) {
+      logger.debug('remote dependency ignored', {dep: scriptDep});
```

Comment:
Currently we're logging what happens to each dependency found via analysis. This has helped a few times with tracking down bad paths, but you're right we could be getting into overkill territory. Lets stick to the trend for now and then reevaluate if logs ever feel too noisy.